From baa77b95538462a65a021e706928d18ad6a5db61 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:51 -0400
Subject: [redhat] kabi: Adding symbol vmalloc_to_page (mm/vmalloc.c)

Message-id: <bf263258affde2fdd728a81a1390392b66445279.1410510773.git.poros@redhat.com>
Patchwork-id: 96124
O-Subject: [PATCH RHEL7.1 231/233] kabi: Adding symbol vmalloc_to_page (mm/vmalloc.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    This function is used to get page for vmalloc'ed address.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/vmalloc_to_page
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/vmalloc_to_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/vmalloc_to_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/vmalloc_to_page b/redhat/kabi/kabi-module/kabi_ppc64/vmalloc_to_page
new file mode 100644
index 0000000..1e0ba95
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/vmalloc_to_page
@@ -0,0 +1,2 @@
+#1-
+0x250a51a0 vmalloc_to_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/vmalloc_to_page b/redhat/kabi/kabi-module/kabi_s390x/vmalloc_to_page
new file mode 100644
index 0000000..0085bb2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/vmalloc_to_page
@@ -0,0 +1,2 @@
+#1-
+0x4333cea3 vmalloc_to_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/vmalloc_to_page b/redhat/kabi/kabi-module/kabi_x86_64/vmalloc_to_page
new file mode 100644
index 0000000..d99486c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/vmalloc_to_page
@@ -0,0 +1,2 @@
+#1-
+0x18e6b5cd vmalloc_to_page vmlinux EXPORT_SYMBOL
-- 
1.7.1